framework/src/org/checkerframework/common/value/ValueTransfer.java

```diff
@@ -108,7 +108,10 @@ public ValueTransfer(CFAbstractAnalysis<CFValue, CFStore, CFTransfer> analysis)
         } else if (isIntRange(subNode, p)) {
             Range range = getIntRange(subNode, p);
             List<Long> longValues = ValueCheckerUtils.getValuesFromRange(range, Long.class);
-            values = NumberUtils.castNumbers(subNode.getType(), longValues);
+            values =
+                    longValues == null
+                            ? null
+                            : NumberUtils.castNumbers(subNode.getType(), longValues);
```

Comment:
I changed NumberUtils.castNumbers to except a null parameter in #1217 (which was only merged earlier today).  So this part of the change isn't needed, but we should add the testcase to make sure.